Increased version number to v0.1.0 and updated changelog (#179)
Adyen/adyen-flutter: README.md

),
``` 

<br>

### Instant Component with Sessions flow
1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
- `sessionData`: the payment session data.
- `id`: a unique identifier for the session data.

> - The request body.

Reviewer: ```suggestion

```

> Pass these values to your app.

Reviewer: ```suggestion
2. Pass these values to your app.
```

2. In your Flutter app, create an `InstantComponentConfiguration`.

```dart

> final InstantComponentConfiguration instantComponentConfiguration = InstantComponentConfiguration(
>   // Change the environment to live when you are ready to accept real payments.
>   environment: Config.environment,
>   clientKey: Config.clientKey,
>   countryCode: Config.countryCode,
>   amount: Config.amount,
> );

Reviewer: Better avoid sudo code. Some people may be confused with `Config` and think it is an object provided by our SDK.

```suggestion
final InstantComponentConfiguration instantComponentConfiguration = InstantComponentConfiguration(
  // Change the environment to live when you are ready to accept real payments.
  environment: Environment.Test,
  clientKey: "{YOUR_CLIENT_KEY}",
  countryCode: "US",
  amount: Amount("USD", 1000), // Amount in minor units
);
```
